tests/test-daemon.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This test daemonizes an nmsgtool and checks its output to make sure it is
+# equivalent to the input file.  It also tests the PID functionality present
+# in daemon.c.
+
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+nmsgtool_test=$script_dir/../src/nmsgtool
+infile=$script_dir/private-tests/202_sample.nmsg
+outfile=/tmp/testdaemon.nmsg
+pidfile=/tmp/testdaemon.pid
+retval=0
+
+check() {
+	if [ $? = "0" ]; then
+		echo "PASS: $*"
+	else
+		echo "FAIL: $*"
+		retval=1
+	fi
+}
+
+killproc () {
```

Comment:
This can be removed. A simple invocation of `kill` should do the trick

Reply:
Fixed